### ISSUE TITLE
ntmdt stage xy movement generates MANY warnings

### ISSUE THREAD

**Issue by mvbnano** [QMAPP-mq/qudi#39]
## *What* is affected by this bug?
There are **constant** warnings displayed each time a movement is made. The warning (see details below) is concerning the z movement of the stage.

## *When* does this occur?
When a xy scan is launched in the confocal, the following warning it **constantly** displayed:
```
warning Cannot make the movement of the z axissince the boarder [1e-07,6e-06] would be crossed! Ignore command!
```

The range [1e-07,6e-06] is the constraints on the z axis.

## *Where* on the platform does it happen?
The error message is being generated inside the hardware module ```ntmdt_piezo_stage.py```.

## Expected behavior (i.e. solution)
It should behave as expected.

## Other Comments
**It does not appear that this is impacting the movement of the x and y axes.**
You can see below that come count date is still being collected.

## Screenshot
![image](https://user-images.githubusercontent.com/13956283/37940224-41fe166c-31b3-11e8-851b-a8cc1cf5dcd8.png)

![image](https://user-images.githubusercontent.com/13956283/37940266-7e1bd756-31b3-11e8-9932-b4ca4acbf122.png)

**Comment by mvbnano** [comment]
The probable cause of this appears to be because the confocal thinks that is is at z = 20um.

![image](https://user-images.githubusercontent.com/13956283/37940669-a2212550-31b5-11e8-98c9-f2fd11bf7580.png)

I am not sure why this is happening. I have ensured that the constraints, for the z axis,  in the configuration are set correctly.

The confocal scan can still be run in the xy plane, I will just not move it in z.

**Comment by mvbnano** [comment]
The stage can successfully scan a single line. However, the below error prevents it scanner the next line.

![image](https://user-images.githubusercontent.com/13956283/37941164-853e03ba-31b8-11e8-8c1a-835b70fd7c53.png)

**Comment by mvbnano** [comment]
Turning it on and off again fixed the single line scan error (I think):

![image](https://user-images.githubusercontent.com/13956283/37941648-f8b0d564-31ba-11e8-8581-98b0cb4d2105.png)

**Comment by latchr** [comment]
The 20 um z position could be due to restored StatusVariables. I would suggest that you clear the app_status directories in the location of your config file. Do this on both the vm and the "master" machine just in case.

You will know if it worked, because when you load the confocal GUI it should show only 0-6um in the z range, and probably put the crosshair at 3um by default (midrange).

**Comment by mvbnano** [comment]
Thanks @latchr , unfortunately this did not do the trick. Odd behaviour.

**Comment by mvbnano** [comment]
Fixed in commit 96feb8404f4ddd6ff9e2110cbc0b39078478256c
